lib/src/sprite.dart

```diff
@@ -76,4 +78,11 @@ class Sprite {
 
     canvas.drawImageRect(image, src, drawRect, drawPaint);
   }
+
+  /// Return a new Image based on the [src] of the Sprite.
```

Comment:
Add a comment here to warn people about this being a a heavy operation and that it shouldn't be called on game render. What do you think?

Reply:
Added it!